argilla-server/src/argilla_server/errors/future/base_errors.py

```diff
@@ -55,6 +57,11 @@ class MissingVectorError(UnprocessableEntityError):
     pass
 
 
+class UpdateDistributionWithExistingResponsesError(UnprocessableEntityError):
+    def __init__(self, message):
+        super().__init__(message, code=UPDATE_DISTRIBUTION_WITH_EXISTING_RESPONSES_ERROR_CODE)
```

Comment:
Why we need this error to have an specific code and being managed in a special way?